b42_prepush_check.sh

```diff
@@ -29,27 +34,30 @@ summary_status="${summary_status}.${uncommited_status}"
 
 echo "== Filename to function name"
 not_found=0
+OLDIFS="${IFS}"
+IFS=$'\n'
 for fname in $(find . -name 'ft_*.c'); do
-	func_name=$(basename -s .c $fname);
+	func_name="$(basename -- "$fname" .c)"
```

Comment:
I prefer to use `-s` for the basename call, as I found it's less "cryptic". and also it makes function safer if at some point it will extended, for example with `xargs` usage.
What are the reasons or cases we should prefer `basename -- name suffix` over `basename -s suffix name`?

Reply:
No, that's a fair point, especially the xargs one!

Reply:
I'm changing it to `basename -s .c -- "$fname"`, leaving the `--` since a folder could begin with a dash (just to make sure...)

Reply:
```suggestion
	func_name="$(basename -s .c -- "$fname")"
```